### PR TITLE
Line wrapping fixes

### PR DESCRIPTION
**Pull request by dericed**
breaks alignment but fixes compliance with chars-per-line limit

### CODE REVIEW

File: specification.markdown [hunk 1]
```diff
@@ -320,7 +320,7 @@ The path defines the allowed storage locations of the `EBML Element` within an `
 The `path` attribute is REQUIRED.
 
 ```
-EBMLFullPath             = EBMLElementOccurrence "(" EBMLReferencePath ")"
+EBMLFullPath           = EBMLElementOccurrence "(" EBMLReferencePath ")"
```
Comment (retokromer) [inline specification.markdown]: LUTM, but fixes the issue (U = ugly ;-)

**Comment by retokromer** [comment]
fixes https://github.com/Matroska-Org/ebml-specification/issues/148

File: specification.markdown (hunk 1 above)
Comment (dericed) [inline specification.markdown]: i can't think of a better idea, unless @robUx4 wants to rename one of these components to use a shorter string length.

Comment (robUx4) [inline specification.markdown]: Unless @retokromer comes with something less U, I think it's fine.

**Comment by robUx4** [comment]
LGTM although it might break the ugly code that's generating various code from the XML source.

File: specification.markdown (hunk 1 above)
Comment (dericed) [inline specification.markdown]: Could also resolve by renaming `VariableParentOccurrence` to `VarParentOccurrence`.

Comment (retokromer) [inline specification.markdown]: No, I don’t have a proposal having less U. As Dave said, only shorter names would prevent it. That said, this fixes the issue of the «miniature pages».

Comment (dericed) [inline specification.markdown]: I think shorter names are less `U` than breaking alignment.

**Pull request merged**